roles/openshift_master_facts/test/openshift_master_facts_bad_input_tests.py

```diff
@@ -0,0 +1,57 @@
+import copy
+import os
+import sys
+
+from ansible.errors import AnsibleError
+import pytest
+
```

Comment:
I'm good with segregating the tests by directory as well.

I don't think we should centralize tests, though. I think bundling the tests with the roles, playbooks, etc they belong to gives us a way to ensure that they can stand on their own. It also makes it easier to match tests to those components for contributors.